examples/globe.js

```diff
@@ -3,18 +3,48 @@
 
 // Define initial camera position
 var positionOnGlobe = { longitude: 2.351323, latitude: 48.856712, altitude: 25000000 };
+var promises = [];
+var miniView;
+var minDistance = 10000000;
+var maxDistance = 30000000;
 
 // `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
 var viewerDiv = document.getElementById('viewerDiv');
+var miniDiv = document.getElementById('miniDiv');
 
 // Instanciate iTowns GlobeView*
 var globeView = new itowns.GlobeView(viewerDiv, positionOnGlobe, { renderer: renderer });
-
-var promises = [];
-
 function addLayerCb(layer) {
     return globeView.addLayer(layer);
 }
+
+if (!renderer) {
```

Comment:
Is this really needed?
If so, maybe add a comment explaining why you do this (I suppose you want to detect if you're in a test env or not?)